packages/dashboard/e2e/tests/dashboard/dashboard.spec.ts

```diff
@@ -145,3 +152,50 @@ test('pagination buttons', async ({ page }) => {
   expect(forwardBtn);
   expect(backBtn);
 });
+
+test('dashboard viewport gestures', async ({ page }) => {
+  await page.goto(TEST_PAGE);
```

Comment:
note to future self: create utility to create dashboard, add line widget, and add property to line widget :) will make writing playwright tests faster

Reply:
Would it be worth making a backlog task for this?

Reply:
good point! just made one :)